docs/build/js-client/3-hello-world.md

```diff
@@ -121,61 +118,58 @@ Using the code generated by the compiler is as easy as calling a function. The c
 Let's see how to use the generated code in our application. The `index.ts` file looks this way:
 
 ```typescript
-import "@fluencelabs/js-client.node"; // Import the JS Client implementation. Don't forget to add this import!
-import { Fluence } from "@fluencelabs/js-client.api"; // Import the API for JS Client
-import { kras } from "@fluencelabs/fluence-network-environment"; // Import list of possible relay nodes (network environment)
+import { 
+    Fluence, // Import the API for JS Client
+    randomKras // // Import function for choosing random relay from list of possible relay nodes (network environment)
```

Comment:
doesn't compile:
```
TSError: ⨯ Unable to compile TypeScript:
src/index.ts:3:5 - error TS2305: Module '"@fluencelabs/js-client"' has no exported member 'randomKras'.
```

Reply:
This should compile with the latest js-client

Reply:
following the instructions, i ended up [with](https://github.com/boneyard93501/jsc-tests). see the readme.md for a couple errors i ran into.

Reply:
There is no readme. Regarding js-client - you're right. To make the example work you need to install js-`client@unstable`. Added a commit with the fix

Reply:
forgot to push, should be there now.
now i get wrong ts target error:
```
TSError: ⨯ Unable to compile TypeScript:
src/index.ts:12:1 - error TS1378: Top-level 'await' expressions are only allowed when the 'module' option is set to 'es2022', 'esnext', 'system', 'node16', or 'nodenext', and the 'target' option is set to 'es2017' or higher.
```
i updated config to below and a few variations:
```
{
  "compilerOptions": {
      "target": "ESNext",
      "module": "ESNext",
      "strict": true,
      "skipLibCheck": true,
      "moduleResolution": "nodenext"
  },
  "ts-node": {
      "esm": true
  }
}
```
but ended up getting: `SyntaxError: Cannot use import statement outside a module`

Reply:
Thanks, i will check everything again and update instruction

Reply:
Added missing instruction to doc.